### PR TITLE
fix: race condition

### PR DESCRIPTION
**Pull request by FelipeMCassiano**
tip: try use -race flag when running to check if there's a race condition (works almost every time

### CODE REVIEW

File: internal/config/config.go [hunk 1]
```diff
@@ -4,23 +4,24 @@ import (
...
+	"github.com/kauefraga/anubis/internal/models"
 	"github.com/pelletier/go-toml/v2"
 )
 
 type Config struct {
 	Version   int
 	Port      uint16
 	Algorithm string
-	Servers   []string
+	Servers   []*models.Servers
```
Comment (kauefraga) [inline internal/config/config.go:15]: How this structure looks like in the TOML file?

File: internal/algorithms/leastconnection.go
Comment (kauefraga) [inline internal/algorithms/leastconnection.go:1]: I think that the mutex solves the problem well here, no need to use atomic operations.

File: internal/algorithms/roundrobin.go
Comment (kauefraga) [inline internal/algorithms/roundrobin.go:1]: Nice! Atomic should increase performance here

**Review by kauefraga — CHANGES_REQUESTED** [review]
Addressing the comments I leave should make it fine

File: internal/config/config.go (hunk 1 above)
Comment (FelipeMCassiano) [reply internal/config/config.go:15]: like this 
[[servers]]
url = "localhost:8080"

[[servers]]
url = "localhost:8081"

Comment (kauefraga) [reply internal/config/config.go:15]: Looks good!

**Review by kauefraga — APPROVED** [review]
Seems nice! Thank you for the this fix.

**Pull request merged**